dev/archery/archery/integration/runner.py

```diff
@@ -435,6 +435,11 @@ def run_all_tests(with_cpp=True, with_java=True, with_js=True,
             description="Ensure Flight SQL protocol is working as expected.",
             skip={"Rust"}
         ),
+        Scenario(
+            "flight_sql:extension",
+            description="Ensure Flight SQL extensions work as expected.",
+            skip={"Rust", "Go"}
```

Comment:
should i wait until this gets merged before implementing this in Go? or should i contribute it to this RFC?

Reply:
It's up to you - though let me take a look at CI here